www/src/utils/typography.js

```diff
@@ -88,22 +87,22 @@ const _options = {
       hr: {
         backgroundColor: colors.ui.light,
       },
+      "tt, code, kbd, samp": {
+        // reset line-height: 1.4rem set by
+        // https://github.com/KyleAMathews/typography.js/blob/3c99e905414d19cda124a7baabeb7a99295fec79/packages/typography/src/utils/createStyles.js#L198
+        lineHeight: `inherit`,
```

Comment:
When putting together the GIF I noticed a slight increase in vertical whitespace when inline `<code>` was rendered with `Menlo`. I tracked it back to `line-height` being set to `1.4rem` for `tt, code, kbd, samp`, which I'm almost sure typography.js should not do. Will open an issue upstream to track this.